Maui.Tabs/Maui.Tabs.csproj

```diff
@@ -1,13 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0;net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net6.0-windows10.0.19041.0</TargetFrameworks>
+		<TargetFrameworks>net9.0</TargetFrameworks>
```

Comment:
It's certainly ok to remove `net6.0`, since it's out of support already (at least regarding MAUI), but shouldn't one also add `net8.0` here, as it currently is the only stable MAUI version with active support?